crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4236,15 +4300,17 @@ impl<F> TryFrom<&AdyenRouterData<&types::PayoutsRouterData<F>>> for AdyenPayoutF
         let payout_type = item.router_data.request.payout_type.to_owned();
         let merchant_account = auth_type.merchant_account;
         match payout_type {
-            storage_enums::PayoutType::Bank => Ok(Self::Bank(PayoutFulfillBankRequest {
-                merchant_account,
-                original_reference: item
-                    .router_data
-                    .request
-                    .connector_payout_id
-                    .clone()
-                    .unwrap_or("".to_string()),
-            })),
+            storage_enums::PayoutType::Bank | storage_enums::PayoutType::Wallet => {
+                Ok(Self::GenericFulfillRequest(PayoutFulfillGenericRequest {
+                    merchant_account,
+                    original_reference: item
+                        .router_data
+                        .request
+                        .connector_payout_id
+                        .clone()
+                        .unwrap_or("".to_string()),
```

Comment:
We can throw an error here, since `original_reference` is a required field for fulfillment